### ISSUE TITLE
Civs and cops should be automatically dropping rifles when they die.

### ISSUE THREAD

**Issue by filonzi** [sync0s/i3t#76]
Civs and cops should be automatically dropping rifles when they die so that they have something to lose.

Otherwise what are they gonna spend most of their money on for now?

**Comment by Spellchk** [comment]
For civs they do if it is on thier shoulder.  It drops and then poofs into thin air.

**Comment by filonzi** [comment]
Even if they get revived? what about if they die while carrying it?

**Comment by Spellchk** [comment]
Moment they die it drops and goes poof.  But only the rifle.

**Comment by skinner904** [comment]
I got killed tonight and didnt lose my rifle because I got revived. That is how its supposed to work I thought.

**Comment by filonzi** [comment]
Yeah but while you are down waiting to get revived your rifle should be on the ground giving your killer the chance to grab it or the cops to confiscate it.
